### ISSUE TITLE
Tasks to fix before MVP

### ISSUE THREAD

**Issue by zhu0619** [polaris-hub/polaris-recipes#2]
**What content related issue can be fixed ?**
- Some dataset readme are missing a clickable link to the original paper or pubchem/chembl assay from which the data is coming from. Can you add it?
- The dataset page should also forward to the notebook that curated the dataset (not just the benchmark page).
    - In the readme, instead of "For more details of this benchmark " can you replace with something like "The full curation and creation process is documented at xxxxxx"
    - Lu: are the baselines script and code documented somewhere? Not high priority for the MVP but we will want to disclose this before the public release.

- General structure — is this right? May be worth including a bit more structure (headers):
    - Background
    - Description of readout
    - Published datasets
- Not a huge fan of “higher the more potent”, and it doesn’t fit with THTB. You could even say ‘Higher potency’ or something?
- There’s enough jargon around it makes me wonder if there couldn’t be a link to a table of acronyms, at least.
- TSNE vs UMAP - consistency? In notebooks it’s a UMAP but in details it’s a TSNE. Only included sometimes, why?

**What is a quick win we can address before MVP**
    - The leaderboard page shows plenty of NaN: can we only show one single metric (the main one)?
    - The detailed results page are missing a link to the "Related benchmark".

**Comment by cwognum** [comment]
Any thoughts on merging datasets and benchmarks, Lu? I feel quite strongly that that could help a lot!

**Comment by cwognum** [comment]
> What is a quick win we can address before MVP
> - The leaderboard page shows plenty of NaN: can we only show one single metric (the main one)?
> - The detailed results page are missing a link to the "Related benchmark".


These are already tracked in https://github.com/polaris-hub/polaris-hub/issues/110 and https://github.com/polaris-hub/polaris-hub/issues/44.

I believe other quick wins for the UI we can tackle would be: 
1. How the modality is visualized, see https://github.com/polaris-hub/polaris-hub/issues/179
2. Move the README to the top of the page. 
3. Make attribute and value bold in the "Explore this dataset" module. 
4. Add tooltips (question mark icon that you hover on for additional info) for the various sections.
5. Increase size of code pop-up for "Try it out" 

More challenging, but worth a shot: 
1. Make model cards in catalog easier to distinguish, e.g. by visualizing tags? Will need some design.

**Comment by zhu0619** [comment]
> Any thoughts on merging datasets and benchmarks, Lu? I feel quite strongly that that could help a lot!

I agree on merging the datasets. But for benchmarks, the single task wide type and the one with mutants are not necessary from the same dataset. I see the confusion there. We can keep the ones with larger data size and keep in one benchmark if there are also mutants from the same dataset.

**Comment by cwognum** [comment]
If the wild type and mutants don't come from the same dataset, could the wild type be used as a test set only? Or is the data generation protocol different? I guess the latter, but wanted to raise the idea nonetheless.
